core/src/main/java/com/airbnb/aerosolve/core/function/Point.java

```diff
@@ -0,0 +1,93 @@
+package com.airbnb.aerosolve.core.function;
+
+import com.airbnb.aerosolve.core.FunctionForm;
+import com.airbnb.aerosolve.core.ModelRecord;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Point implements Function {
+  private float weight;
+
+  public Point(float weight) {
+    this.weight = weight;
+  }
+
+  public Point(ModelRecord record) {
+    List<Double> weightVec = record.getWeightVector();
+    weight = weightVec.get(0).floatValue();
+  }
+
+  @Override
+  public Function aggregate(Iterable<Function> functions, float scale, int numBins) {
```

Comment:
not for this PR, but `numBins` is not used, I feel we may want to make `numBins` available only to Spline related features, otherwise, whenever we add new function type, we have to add this.